ci-operator/config/openshift/machine-config-operator/openshift-machine-config-operator-master.yaml

```diff
@@ -15,13 +15,26 @@ images:
 promotion:
   name: "4.3"
   namespace: ocp
+raw_steps:
+- rpm_image_injection_step:
+    from: machine-config-operator
```

Comment:
Why are you basing this on the mco image?  Your contents should be depending on the source image.

Reply:
fixed, thanks!